core/vm/interpreter.go

```diff
@@ -187,10 +188,13 @@ func (in *EVMInterpreter) Run(contract *Contract, input []byte, readOnly bool) (
 		op = contract.GetOp(pc)
 		operation := in.cfg.JumpTable[op]
 		// Validate stack
-		if sLen := stack.len(); sLen < operation.minStack {
+		sLen := stack.len()
+		if sLen == int(params.StackLimit) {
```

Comment:
The slowdown is possibly because I put `sLen` into the outer scope.
```suggestion
		if sLen := stack.len(); sLen == int(params.StackLimit) {
```

Reply:
Still slow after the fix.